sphinx/ext/intersphinx/_load.py

```diff
@@ -130,7 +199,7 @@ def fetch_inventory_group(
                     return True
         return False
     finally:
-        if failures == []:
+        if not failures:
```

Comment:
This shouldn't cause any problems, right?  (`failures` is _definitely_ a `list`, and even if this causes bytecode cache invalidation, those caches are per-module anyway, and other changes in this file would cause that anyway)

Reply:
This is fine, it's a style point (I sometimes prefer doing checks like this or like ``len(obj) == 0`` as it is a quick reminder what *kind* of truth testing you're doing.